frontend/src/store/modules/actuator.ts

```diff
@@ -1,13 +1,21 @@
 import axios from "axios";
 import { defineStore } from "pinia";
-import { ActuatorState, ServerInfo } from "@/types";
+import { ActuatorState, ServerInfo, Release } from "@/types";
+import { useLocalStorage } from "@vueuse/core";
+import { semverCompare } from "@/utils";
 
 const EXTERNAL_URL_PLACEHOLDER =
   "https://www.bytebase.com/docs/get-started/install/external-url";
+const GITHUB_API_LIST_BYTEBASE_RELEASE =
+  "https://api.github.com/repos/bytebase/bytebase/releases";
```

Comment:
Is it accessible for non-global clients?

Reply:
Yes it is